packages/react-text/src/components/Title3/Title3.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import { Title3Props } from './Title3.types';
+import { renderText, useText, useTextStyles } from '../Text/index';
+import { typographyStyles } from '../../index';
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: typographyStyles.title3,
+});
+
+/**
+ * Text wrapper component for the Title 3 typography variant
+ */
+export const Title3 = React.forwardRef<HTMLElement, Title3Props>((props, ref) => {
```

Comment:
all of these wrappers have the same implementation. 

cant we make it DRY
- less javascript for consumers
- easier to maintain

I quickly prototyped something like this:

```ts
// @filename: src/components/wrapper.tsx

import * as React from 'react';
import { mergeClasses } from '@fluentui/react-make-styles';
import { renderText, TextProps, useText, useTextStyles } from '../Text';

export interface Props extends Omit<TextProps, 'font' | 'size'> {}

export function createWrapper(options: { displayName: string; useStyles: () => Record<'root', string> }) {
  const { useStyles, displayName } = options;
  const Wrapper = React.forwardRef<HTMLElement, Props>((props, ref) => {
    const styles = useStyles();
    const state = useText(props, ref);
    useTextStyles(state);

    state.className = mergeClasses(state.className, styles.root, props.className);

    return renderText(state);
  });
  Wrapper.displayName = displayName;

  return Wrapper;
}
```

```ts
// @filename src/components/Title3.tsx
import { makeStyles } from '@fluentui/react-make-styles';
import { typographyStyles } from '../../index';

import { createWrapper } from '../wrapper';

/**
 * Styles for the root slot
 */
const useStyles = makeStyles({
  root: typographyStyles.title3,
});

/**
 * Text wrapper component for the Title 3 typography variant
 */
export const Title3 = createWrapper({ useStyles, displayName: 'Title3' });
```

- test are passing
![2021-07-16 at 15 36](https://user-images.githubusercontent.com/1223799/125956582-858bc41a-bfc1-4166-bacc-f9cec36d7e33.png)